plugins/workbench-resources/src/components/Workbench.svelte

```diff
@@ -300,6 +302,9 @@
     if (loc.path.length > 3 && getSpecialComponent(loc.path[3]) === undefined) {
       // resolve short links
       const resolvedLoc = await resolveShortLink(loc)
+      if(locationResolver === "tracker:resolver:Location" && resolvedLoc === undefined) {
```

Comment:
Not good, workbench should not know about tracker at all.

Reply:
Okay, will try to update the condition in `tracker-resources`

Reply:
Kind of stuck. Currently, if we move to an undefined issue URL, so by default, `SpaceView` would be rendered as rest are undefined. And, in `SpaceView`, no content would be rendered as the stuff that it needs are undefined.